src/api/Ethereum.ts

```diff
@@ -178,6 +203,26 @@ export const apiForCurrency = (currency: CryptoCurrency): API => {
       return data;
     },
 
+    async getNFTMetadata(input) {
+      const { data }: { data: NFTMetadata[] } = await network({
+        method: "POST",
+        /** @important FIXME: metadata are mocked while we are waiting for the API */
+        url: "https://mock-nft-metadata-api.vercel.app/v1/chains/eth/contracts/tokens/infos",
+        data: input,
+      });
+
+      return data.reduce(
+        (acc, curr) =>
+          _.setWith(
```

Comment:
could you not use lodash for simple things like this :grin: 
But if you really need it cause its easier for you try and import just the function you use such as this
```javascript
import setWith from "lodash/setWith";
```

Reply:
Yeah that's just a one line solution (if prettier didn't force the new lines) to the infamous
```javascript
data.reduce((acc, curr) => {
  const contract = curr?.contract?.toLowerCase();
  const tokenId = curr?.tokenId?.toLowerCase();

  if (!acc[contract]) {
    acc[contract] = {};
  }

  acc[contract][tokenId] = curr;
}, {});
```

I can put that if you prefer 😄 

But yeah if you're ok with Lodash, I'll use the tree shaking 👌